xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,508 @@
+            <remark>toms 2020-05-14: See also bsc#1171681</remark>
+            <para>
+               An additional machine which will become the &qnet; server with
+               &sls; and <package>corosync-qnetd</package> package installed.
+            </para>
+         </listitem>
+         <listitem>
+            <para>
+               A network address used by the &qnet; server which can be neutrally
+               reached by all cluster nodes.
+            </para>
+         </listitem>
+      </itemizedlist>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-setup-qnetd">
+      <title>Setting up the &qnet; Server</title>
+      <para>
+         The &qnet; server is not part of the cluster stack, it is also
+         not a real member of your cluster. As such, you cannot move resources
+         to this server.
+      </para>
+      <para>
+         As the &qnet; server is <quote>state free</quote>, you do not usually
+         need to configure anything (although you could in <filename>/etc/sysconfig/corosync-qnetd</filename>).
+         The <package>corosync-qnetd</package> service by default runs the daemon
+         as user <systemitem class="username">coroqnetd</systemitem>
+         in group <systemitem class="groupname">coroqnetd</systemitem>. This avoids
+         to run the daemon as &rootuser;.
+      </para>
+      <para>
+         To create a &qnet; server, proceed as follows:
+      </para>
+      <procedure>
+         <step>
+            <para>
+               On the machine that will become the  &qnet; server, install
+               &sls; &productnumber;.
+               <remark>toms 2020-05-15: See also bsc#1171681</remark>
+            </para>
+         </step>
+         <step>
+            <para>
+               Log in to the &qnet; server and install the following package:
+            </para>
+            <screen>&prompt.root;<command>zypper</command> install corosync-qnetd</screen>
+         </step>
+         <step>
+            <para>
+               Start and enable the service:
+            </para>
+            <screen>&prompt.root;<command>systemctl</command> enable -now corosync-qnetd</screen>
+         </step>
+      </procedure>
+      <para>
+         Your &qnet; server is ready to accept connections from a &qdevice; client
+         <systemitem>corosync-qdevice</systemitem>.
+         Further configurations are not needed.
+      </para>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-qdevice">
+      <title>Connecting &qdevice; Clients to the &qnet; Server</title>
+      <para>
+         After you have set up your &qnet; server, we are ready to set up
+         and run the clients.
+         You can connect the clients during the installation of your cluster
+         or you can add them later. In the following procedure we use
+         the latter approach. We assume two cluster nodes (&node1; and &node2;)
+         which are already part of a two-node cluster and the &qnet; server (&node3;).
```

Comment:
"which are already part of a two-node cluster" is contradictory to the first two steps where the cluster is set up...

Reply:
I rephrased the sentence and use:

```xml
We assume a cluster with two cluster nodes (&node1; and &node2;) and
the &qnet; server (&node3;).
```